src/_scss/pages/award/shared/awardAmountsSection/_dataTable.scss

```diff
@@ -5,7 +5,8 @@
         @extend %column-less-table__row;
         flex-wrap: wrap;
         border-bottom: 0;
-        white-space: nowrap;
+        // white-space: nowrap;
```

Comment:
Did you leave this in intentionally or should it be deleted?

Reply:
nope, my mistake